NOTICE.txt

```diff
@@ -1876,7 +1876,7 @@ SOFTWARE.
 This product contains a modified portion of 'gorp',  to save time, minimize the drudgery of getting data in and out of the database, and help code focus on algorithms, not infrastructure by James Cooper.
 
 * HOMEPAGE:
-  * https://github.com/go-gorp/gorp
+  * https://github.com/mattermost/gorp
```

Comment:
this is still `https://github.com/go-gorp/gorp` we just forked it to fix a few issues.